llvm/test/Transforms/Inline/promote-call-bitwidth.ll

```diff
@@ -0,0 +1,46 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --function-signature --scrub-attributes
+; RUN: opt -S -passes=inline < %s | FileCheck %s
+target datalayout = "e-m:e-p200:128:128:128:64-p:64:64-i64:64-i128:128-n64-S128-A200-P200-G200"
```

Comment:
```suggestion
target datalayout = "p200:128:128:128:64-A200-P200-G200"
```
I think this should be sufficient (and I'm not sure we even need the default address space numbers, but good to keep them.

Reply:
Done